lunar/distribution.yaml

```diff
@@ -2059,6 +2059,20 @@ repositories:
       url: https://github.com/ros/ros_tutorials.git
       version: lunar-devel
     status: maintained
+  ros_type_introspection:
+    doc:
+      type: git
+      url: https://github.com/facontidavide/ros_type_introspection.git
+      version: master
+    release:
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/facontidavide/ros_type_introspection-release.git
+      version: 0.6.0-0
+    source:
+      type: git
+      url: https://github.com/facontidavide/ros_type_introspection.git
+      version: master
```

Comment:
@facontidavide I noticed that there is no status for this package on lunar, while it's "developed" on every other distro. Is it on purpose ?